### DISCUSSION TITLE
Serving the github pages under the original URL using a DNS redirect

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#9] [Announcements]
Sorry, initially posted in the wrong repo.  This discussion goes here.
I moved the URL pointer [https://cosmologystatement.org](https://cosmologystatement.org), now it goes to the GitHub ACG webpage.
This works fine!

**I don't have access to custom domain configuration,** there is no *Custom Domain* option under *Pages*.  To me, it looks like this:
![pages-no-custom-domain](https://github.com/a-cosmology-group/cg/assets/16748402/b870d447-92a9-4ad1-b6ce-bc29f9098640)

.
**-----------It should look like this!----------**
![pages-custom-domain](https://github.com/a-cosmology-group/cg/assets/16748402/aabd80e8-d121-4721-a844-f47a79afc11e)

**I promise I won't touch anything else!** :-)

**Comment by ftessier** [comment]
Sorry I did not make you admin, only maintainer. It is fixed now, you are admin.
